src/plib/resources/locales/en-US.php

```diff
@@ -15,6 +15,7 @@
     'authDataSaved' => 'The authorization data was saved.',
     'getAuth' => 'Find your root account credentials here',
     'getAuthStepTwo' => 'You can also create one yourself. If you do, make sure to grant the "route53:*" and "route53domains:*" permissions to the account (<a href="%%learnMoreUrl%%">learn more</a>).',
+    'getAuthNoRoot' => 'Please, make sure to grant the "route53:*" and "route53domains:*" permissions to the account (<a href="%%learnMoreUrl%%">learn more</a>).',
```

Comment:
I believe that this is some kind of an error message? In what situation does it appear?

Reply:
![image](https://github.com/plesk/ext-route53/assets/51955156/de467f29-331b-443b-b825-e2bd8cb4da97)
I think it should help

Reply:
So, it`s refer to AWS account

Reply:
Make sure that <a href="%%learnMoreUrl%%">you have granted the "route53:*" and "route53domains:*" permissions</a>  to your IAM user account.